packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/RealmMapInternal.kt

```diff
@@ -61,7 +63,7 @@ internal abstract class ManagedRealmMap<K, V> constructor(
     internal val parent: RealmObjectReference<*>,
     internal val nativePointer: RealmMapPointer,
     val operator: MapOperator<K, V>
-) : AbstractMutableMap<K, V>(), RealmMap<K, V>, CoreNotifiable<ManagedRealmMap<K, V>, MapChange<K, V>>, Flowable<MapChange<K, V>> {
```

Comment:
why removing `Flowable` interface?

Reply:
It was an internal interface that matched our public API. But with the KeyPath changes that is no longer the case and is actually harmful instead now as Notifier cannot use it to handle all flows the same.